### DISCUSSION TITLE
Does VPF have the ability to read images or a directory of images?

### DISCUSSION THREAD

**Discussion by avickars** [NVIDIA/VideoProcessingFramework#465] [Ideas]
**Is your feature request related to a problem? Please describe.**

Small question, does VPF have the ability to read images or a directory of images similar to what gstreamer is capable of doing?

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by theHamsta** [comment]
VPF does not have this ability at the moment. We're considering this feature for a next iteration of VPF.
